meta/sai_meta.cpp

```diff
@@ -6024,6 +6063,35 @@ void meta_sai_on_fdb_event_single(
 
     std::string key_fdb = sai_serialize_object_meta_key(meta_key_fdb);
 
+    /*
+     * Because we could receive fdb event's before orch agent will query or
+     * create bridge/vlan/bridge port we should snoop here new OIDs and put
+     * them in local DB.
```

Comment:
The comment is a little confusing.
In what scenario  fdb event may be received before orch agent  "create" bridge/vlan/bridge port?  If that does happen,  the VID that comes with FDB will be different from VID assigned to the object during "create" operation.  The integrity of meta reference will be messed up.

Reply:
there are bridge ports creted by default on the switch and between create switch and removing them it may happen that fdb_event arrive

Reply:
Ok,  I thought you were talking about FDB event for the non-default bridge/vlan/bridge port object.

If it does happen that fdb_event arrive before removing default bridge ports, will it cause validation error since the bridge ports has non-zero reference count? Orchagent initialization fails at this case?